plugins/lookup/etcd3.py

```diff
@@ -218,12 +218,12 @@ def run(self, terms, variables, **kwargs):
                         if val and meta:
                             ret.append({'key': to_native(meta.key), 'value': to_native(val)})
                 except Exception as exp:
-                    display.warning(f'Caught except during etcd3.get_prefix: {to_native(exp)}')
+                    display.warning(f'Caught except during etcd3.get_prefix: {exp}')
             else:
                 try:
                     val, meta = etcd.get(term)
                     if val and meta:
                         ret.append({'key': to_native(meta.key), 'value': to_native(val)})
```

Comment:
Maybe `str(meta.key)` and `str(val)` here as well?

Reply:
Same in line 219 above

Reply:
Same as above, `to_native()` also converts byte strings to unicode strings, which is very different from what `str()` does.